src/Features/CSharp/Portable/CodeRefactorings/SyncNamespace/CSharpChangeNamespaceService.cs

```diff
@@ -403,7 +403,7 @@ static bool ContainsNamespaceDeclaration(SyntaxNode node)
                 .OfType<NamespaceDeclarationSyntax>().Any();
         }
 
-        private static string? GetAliasQualifier(SyntaxNode name)
+        private static string? GetAliasQualifier(SyntaxNode? name)
```

Comment:
>? [](start = 59, length = 1)

It looks like callers pass non-null always.

Reply:
Consider the call on line 144 where `oldNode` is passed after being assigned a `.Parent` value.

---
In reply to: [362865190](https://github.com/dotnet/roslyn/pull/40699#discussion_r362865190) [](ancestors = 362865190)